package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "honeycomb-web-toolkit",
-  "version": "11.0.5",
+  "version": "11.0.6",
```

Comment:
I'd bump this by a major version, as it looks like it'll be a breaking change - trying out on the website there are some places where we'd want to update the padding on forms when we update to this version (homepage search box, the filters on /hub/events/, some of the non-marketo forms like /hub/entrypage/pass)